pixiedust/display/chart/options/optionsShell.py

```diff
@@ -81,7 +82,7 @@ def get_option_index(self, optid=None, widget=None):
     def main_screen(self):
         self.fieldNamesAndTypes = self.get_field_names_and_types(True, True)
         self.fieldNames = self.get_field_names(True)
-        self._addHTMLTemplate("optionsshell.html")
+        self._addHTMLTemplate("optionsshell.html", iteritems=iteritems)
```

Comment:
@vabarbosa I think it would be simpler to use the iteritems filter instead of passing the function as an argument.